.github/workflows/dolfin-tests.yml

```diff
@@ -39,6 +39,13 @@ jobs:
           python3 -m pip install git+https://github.com/FEniCS/ufl.git
           python3 -m pip install git+https://github.com/FEniCS/ffcx.git
 
+      - name: Run cpp demos
```

Comment:
Is this intentionally in the DOLFINx tests workflow? Would it make more sense in pythonapp.yml?

Reply:
pythonapp.yml installs basix with pip, so it's not installed in the default foder.